packages/planner/src/app/components_ngrx/iterations-modal/iterations-modal.component.ts

```diff
@@ -163,12 +163,16 @@ export class FabPlannerIterationModalComponent implements OnInit, OnDestroy, OnC
     }
   }
 
-  ngOnChanges() {}
+  ngOnChanges() { }
 
   ngOnDestroy() {
     // prevent memory leak when component is destroyed
   }
 
+  get isNameFieldEmpty(): boolean {
+    return this.iteration.name.trim().length <= 0;
```

Comment:
Is there a reason you didn't use `<==` instead?

Reply:
It does not exist :P

Reply:
:face_palm: